### DISCUSSION TITLE
Welcome to Digital Slide Archive and HistomicsTK Discussions

### DISCUSSION THREAD

**Discussion by manthey** [DigitalSlideArchive/digital_slide_archive#286] [Announcements]
👋 Welcome
--------------
We’re using Discussions as a place to connect with other members of our community. We hope that you:

- Ask questions you’re wondering about.
- Share ideas.
- Engage with other community members.
- Welcome others and are open-minded.
- Discuss the direction and future of the HistomicsTK ecosystem.

**Please note**
 If you are looking for help with [Girder](https://github.com/girder/girder) please use the [Girder Discussion Board](https://github.com/orgs/girder/discussions).
 Not sure? Please post here and we will be happy to help.

**Comment by 1liuchunlong** [comment]
![image](https://github.com/user-attachments/assets/ec9f8266-522a-4ba2-8eeb-26ea58e48f1f)
How could I slove this problem

**Comment by dgutman** [comment]
Looking through the error, the openslide wheel is not available for
whatever linux distro you're using.  Try to install openslide directly, and
then reinstall the large_image.

Also, how were you installing this?

If you are using linux, this install should simplify things and download
the correct wheels.

pip install large-image[all] --find-links
https://girder.github.io/large_image_wheels


On Sat, Dec 21, 2024 at 2:52 AM 1liuchunlong ***@***.***>
wrote:

> image.png (view on web)
> <https://github.com/user-attachments/assets/ec9f8266-522a-4ba2-8eeb-26ea58e48f1f>
> How could I slove this problem
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DigitalSlideArchive/digital_slide_archive/discussions/286#discussioncomment-11636124>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFODTX7ZBNBMKB3W4XFTOD2GUM5XAVCNFSM6AAAAABUAMN3A6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTCNRTGYYTENA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <DigitalSlideArchive/digital_slide_archive/repo-discussions/286/comments/11636124
> @github.com>
>


-- 
David A Gutman, M.D. Ph.D.
Associate Professor of Pathology
Emory University School of Medicine
